unifi/Dockerfile

```diff
@@ -1,21 +1,39 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/ubuntu-base:8.2.0
+ARG BUILD_FROM=ghcr.io/hassio-addons/ubuntu-base:9.0.3
 # hadolint ignore=DL3006
 FROM ${BUILD_FROM}
 
+ARG BUILD_ARCH=amd64
+
 # Set shell
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Setup base system
+
+# mongodb-org-server is installed from the community servers as it's gone from
+# repositories for newer ubuntu versions.
+#
+# Beware of the licence change, previously the addon used mongodb version 3.6.9
+# released under the AGPL, while 4.0.3 and newer are released under the non-free SSPL
+# https://en.wikipedia.org/wiki/MongoDB#Licensing
```

Comment:
This stuff should be in the PR description, not source.

Reply:
It's also in one of the commit log, and mentioned with less details in the PR description. I've removed these comments, and modified the PR description.

Reply:
Unresolving this comment, as the above-requested change wasn't made fully.

Reply:
Must have been a rebase mistake, it's removed now